README.md

```diff
@@ -6,7 +6,36 @@ Exercism exercises in Haskell
 
 ## Contributing Guide
 
-Please see the [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md)
+Haskell is a great language! (no, really!). Here in the Haskell track we do not
+just want to help you learn this great language, but we have the ambition to raise
+and share the passion for it. So we very much appreciate your help to make this happen!
```

Comment:
I want to share the passion about the language too, but maybe when people reach this file they are looking for more practical information about how the repository is structured and how to contribute. :smile: